governance/votes/Executive vote - July 4, 2024.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Template - [Executive Vote] Out-of-Schedule Executive Vote - Approve LITE-PSM-USDC DAO Resolution - July 04, 2024
+summary: Approving a DAO Resolution pertaining to the existing Coinbase Custody arrangement (RWA014-A) and the new Lightweight Peg Stability Moodule (LITE-PSM-A). 
+date: 2024-07-04T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Out-of-Schedule Executive Vote - Approve LITE-PSM-USDC DAO Resolution - July 04, 2024
+
+The Governance Facilitators and Dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The "Approve LITE-PSM-USDC" DAO Resolution will be approved.
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Approve "Approve LITE-PSM-USDC" DAO Resolution
+
+- **Authorization:** [Forum post]()
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/coinbase-web3-wallet-legal-overview/24577)
+
+If this executive proposal passes, the following DAO Resolution pertaining to the approval of LITE-PSM-USDC in the context of RWA014-A ([Coinbase Custody arrangement](https://forum.makerdao.com/t/coinbase-custody-legal-assessment/20384)) and the transition from the current custodial solution to a new non-custodial solution that leverages the Coinbase Web3 Wallet (the “W3W”) and the Lightweight Peg Stability Module ([LitePSM](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512)) will be approved:
```

Comment:
I would make the elements of DAO Resolution very clear here. My proposal would be to formulate the Proposal Details as follows:
"If this executive proposal passes, LITE-PSM-USDC DAO Resolution will be approved, triggering the following actions:
- [Legal Agreements](https://forum.makerdao.com/t/coinbase-web3-wallet-legal-overview/24577) will enter into effect.
- Lightweight Peg Stability Module (LITE-PSM-USDC-A/ LitePSM) will be implemented with defined [features and parameters](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512)
- Migration from PSM-USDC-A to LITE-PSM-USDC-A will be triggered, following the defined [migration and offboarding plan](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512)